src/libraries/System.Security.Cryptography.Xml/ref/System.Security.Cryptography.Xml.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
-    <Nullable>disable</Nullable>
+    <Nullable>enable</Nullable>
```

Comment:
enable is the default now
```suggestion
```

Reply:
Why is this resolved? `<Nullable>enable</Nullable>` line is still yet to be removed